src/web/routes/application/are-you-pregnant/are-you-pregnant.js

```diff
@@ -7,6 +8,25 @@ const exampleDate = (fromDate = new Date()) => {
   return `${future.getDate()} ${future.getMonth() + 1} ${future.getFullYear()}`
 }
 
+const contentSummary = (req) => {
+  const pregnantSummary = {
+    key: req.t('areYouPregnant.summaryKey'),
+    value: req.t(req.session.claim.areYouPregnant)
+  }
+  const dueDateSummary = {
+    key: req.t('areYouPregnant.expectedDeliveryDateSummaryKey'),
+    value: [
+      req.session.claim['expectedDeliveryDate-day'],
```

Comment:
You'll need to change this to use the new date formatting.

Reply:
done